style.css

```diff
@@ -1,18 +1,33 @@
-body {
-    padding: 10px 15px;
-    display: flex;
+.header {
+    font-weight: bold;
+    padding-left: 10px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    background: #07889B;
+    color: #FFF;
+}
+.title {
+    font-size: 30px;
 }
 
 div.input {
-    width: calc(100% - 17em);
+    padding: 10px;
+    width: calc(100% - 17em - 10px);
 }
 
 div.damageTable {
     position: fixed;
-    right: 0;
+    right: 10px;
+    top: 130px;
```

Comment:
Yeah, I lined it up before I moved the ele resistance section and resizing the header. ><